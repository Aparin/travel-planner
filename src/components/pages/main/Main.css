#leftGroup {
    width: 400px;
    display: inline-block;
    vertical-align: top;
}

#msg {
    color: red;
    margin: 40px;
    font-size: 24px;
}