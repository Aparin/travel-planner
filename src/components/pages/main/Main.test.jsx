/* eslint-disable no-undef */
import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Main from './Main';

describe('Main', () => {
  it('renders Main without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Main />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  test('Main snapshot', () => {
    const component = renderer.create(
      <Main />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
