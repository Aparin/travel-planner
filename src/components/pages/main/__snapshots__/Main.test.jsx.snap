// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Main snapshot 1`] = `
Array [
  <div
    id="leftGroup"
  >
    <input
      className="input"
      id="NewPlaceName"
      onChange={[Function]}
      onKeyUp={[Function]}
      placeholder="Введите название места и нажмите 'Enter'"
      type="text"
      value=""
    />
  </div>,
  <div
    style={
      Object {
        "display": "inline-block",
        "height": "500px",
        "width": "800px",
      }
    }
  />,
]
`;
